### DISCUSSION TITLE
Logical Properties and Directional Properties

### DISCUSSION THREAD

**Discussion by sadeghbarati** [unovue/shadcn-vue#470] [Polls]
- Logical Properties like: `ps-4`, `padding-inline-start: 1rem`
- Directional Properties like `pl-4`, `padding-left: 1rem;`

Related issue https://github.com/radix-vue/shadcn-vue/issues/357
https://caniuse.com/css-logical-props

**Comment by jd-solanki** [comment]
Everyone is adopting the logical properties including major frameworks like Vuetify. So, I guess we should not use directional properties anymore.

Using logical properties allows us to write less code due to logical nature and it's much cleaner and less confusing

**Reply by rainbowdog1** [reply]
Actually, I think that for example pl-1 is less confusing, because it has the direction of the applying style, so in the example I know for a fact that it is going to apply a padding on to the left side.
